lib/manageiq/appliance_console/message_configuration_server.rb

```diff
@@ -0,0 +1,224 @@
+require "awesome_spawn"
+require "fileutils"
+require "linux_admin"
+require "manageiq/appliance_console/message_configuration"
+
+module ManageIQ
+  module ApplianceConsole
+    class MessageServerConfiguration < MessageConfiguration
+      attr_reader :server_hostname, :jaas_config_path,
+                  :server_properties_path, :server_properties_sample_path,
+                  :ca_cert_path, :ca_key_path, :cert_file_path, :cert_signed_path,
+                  :installed_files
+
+      def initialize(options = {})
+        super(options)
+
+        @server_hostname = my_hostname
+
+        @jaas_config_path                  = config_dir_path.join("kafka_server_jaas.conf")
+        @server_properties_path            = config_dir_path.join("server.properties")
+        @server_properties_sample_path     = sample_config_dir_path.join("server.properties")
+
+        @ca_cert_path                      = keystore_dir_path.join("ca-cert")
+        @ca_key_path                       = keystore_dir_path.join("ca-key")
+        @cert_file_path                    = keystore_dir_path.join("cert-file")
+        @cert_signed_path                  = keystore_dir_path.join("cert-signed")
+
+        @installed_files = [tools_log4_properties_path, jaas_config_path,
+                            client_properties_path, server_properties_path,
+                            messaging_yaml_path, LOGS_DIR, keystore_dir_path]
+      end
+
+      def ask_questions
```

Comment:
Is there a difference between this method and https://github.com/ManageIQ/manageiq-appliance_console/pull/130/files#diff-b896847105d5f7861df9e75f19292936e53103942ce4a0187fa37cafd3671839R25 ? Should they be merged into the base class?

Reply:
Good catch! Thanks!

Addressed by commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/130/commits/13470ad9c2c671cea76655a630e7c0752dee8e4e